### PR TITLE
adding commenting to manuscript_figures.py

### PR DESCRIPTION
**Pull request by corinneteeter**
Please merge these comments (no code changes).

### CODE REVIEW

File: analyses/manuscript_figures.py [hunk 1]
```diff
@@ -126,17 +134,46 @@ def fail_rate(response_list, sign, peak_t):
...
+    Parameters
+    ----------
+    response_list : list of neuroanalysis.data.TraceView objects
+        neuroanalysis.data.TraceView object contains waveform data. 
+        #TODO: is there a place we can direct someone to for reference?
+        
+    Returns
+    -------
+    bsub_mean : neuroanalysis.data.Trace object
+        averages the ephys waveform data in the input response_list TraceView objects and replaces the .t0 value with 0.
```
Comment (campagnola) [inline analyses/manuscript_figures.py]: Also baseliune-subtracts

**Comment by corinneteeter** [comment]
I updated and pushed changes based on your comments.

File: analyses/manuscript_figures.py [hunk 2]
```diff
@@ -126,17 +133,47 @@ def fail_rate(response_list, sign, peak_t):
...
-        trace.t0 = 0
-    avg_trace = TraceList(response_list).mean()
-    bsub_mean = bsub(avg_trace)
+# doc string commented out to discourage code reuse given the change of values of t0
+#    """
+#    Parameters
+#    ----------
+#    response_list : list of neuroanalysis.data.TraceView objects
+#        neuroanalysis.data.TraceView object contains waveform data. 
+#        #TODO: is there a place we can direct someone to for reference?
```
Comment (campagnola) [inline analyses/manuscript_figures.py]: remove?

**Comment by corinneteeter** [comment]
No I don't think the comment should be removed.  It is documenting what the function is doing and will not be parsed as a doc string as you wanted.

________________________________

**Comment by campagnola** [comment]
Looks good to me. Ready for merge?

**Comment by corinneteeter** [comment]
Yup go ahead.  Thanks:)

________________________________

**Comment by corinneteeter** [comment]
PS I figured out that you meant the #TODO, not the entire comment.

________________________________

**Pull request merged**